fe/fe-core/src/test/java/org/apache/doris/common/FeNameFormatTest.java

```diff
@@ -55,6 +55,9 @@ public void testCheckColumnName() {
         ExceptionChecker.expectThrows(AnalysisException.class, () -> FeNameFormat.checkCommonName("fakeType", "_commonName"));
         ExceptionChecker.expectThrowsNoException(() -> FeNameFormat.checkCommonName("fakeType", "common-Name"));
         ExceptionChecker.expectThrowsNoException(() -> FeNameFormat.checkCommonName("fakeType", "commonName-"));
+
+        // check success file name prefix
+        ExceptionChecker.expectThrowsNoException(() -> FeNameFormat.checkColumnName("_success"));
```

Comment:
```suggestion
        // check success file name prefix
        ExceptionChecker.expectThrowsNoException(() -> FeNameFormat.checkSuccessFileName("_success"));
```
Is this wrong here?It should be `checkSuccessFileName`？